src/components/ProfileTree/ProfileTree.js

```diff
@@ -156,6 +157,7 @@ export const ProfileTree = memo(({
                 onUnlink={unlink}
                 onDelete={remove}
                 okapi={okapi}
+                showAsHotLink={showAsHotLink}
```

Comment:
The same for the `ProfileBranch` component, for the `ProfileLabel` it can stay as it is